updated missing server functions

Loki214 opened the pull request.

Loki214 committed:
updated missing functions

Loki214 committed:
Update server.ts

Subtixx commented on src/defs/server.ts:
```diff
@@ -1265,98 +1265,756 @@ tmpDef.argDescs = {};
+tmpDef.label = "aclListRights";
+tmpDef.description = " This function returns a table of all the rights that a given ACL has.";
+tmpDef.returnType = "table";
+tmpDef.args = ["acl theACL, string allowedType"];
```
args is an array and every arg should be one element in the array. Please change it everywhere when there are multiple arguments :/
```ts
["ACL theAcl", "string allowedType"]
```

Subtixx commented on src/defs/server.ts:
```diff
@@ -1265,98 +1265,756 @@ tmpDef.argDescs = {};
+tmpDef.label = "aclRemoveRight";
+tmpDef.description = " This function removes the given right (string) from the given ACL.";
+tmpDef.returnType = "bool";
+tmpDef.args = ["acl theAcl, string rightName "];
```
See Line 1326

Subtixx commented on src/defs/server.ts:
```diff
@@ -1265,98 +1265,756 @@ tmpDef.argDescs = {};
+tmpDef.label = "aclGroupAddObject";
+tmpDef.description = " This function adds an object to the given ACL group.";
+tmpDef.returnType = "bool";
+tmpDef.args = ["aclgroup theGroup, string theObjectName"];
```
See Line 1326

Subtixx commented on src/defs/server.ts:
```diff
@@ -1265,98 +1265,756 @@ tmpDef.argDescs = {};
+tmpDef.label = "aclGroupRemoveACL";
+tmpDef.description = " This function removes the given ACL from the given ACL group. (Please NOTE: The resource that's using this function needs the right to remove an acl.)";
+tmpDef.returnType = "bool";
+tmpDef.args = ["aclgroup theGroup, acl theACL "];
```
See Line 1326

Subtixx commented on src/defs/server.ts:
```diff
@@ -1265,98 +1265,756 @@ tmpDef.argDescs = {};
+tmpDef.label = "aclGroupRemoveObject";
+tmpDef.description = " This function removes the given object (string) from the given ACL group. The object can be a resource or a player. See aclGroupAddObject for more details.";
+tmpDef.returnType = "bool";
+tmpDef.args = ["aclgroup theGroup, string theObjectString"];
```
See Line 1326

Subtixx reviewed (changes requested):
First: thanks for taking the time to make this pull-request. Unfortunately I cannot merge this yet since you made a little mistake in a lot of the definitions 😔 
I haven't marked all occurrences though due to time constraints and being on phone. But that should give you an idea.

Loki214 committed:
arg table fix

Loki214 committed:
done

My bad i forgot that i was using something i made to extract the data from the page for those acl functions

Loki214 committed:
Merge branch 'master' of https://github.com/Loki214/vscode-mtalua

Subtixx commented:
Thanks
